Задание 2/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Задание_2номер_9
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите длину стороны треугольника и прилежащие к ней углы");
+            //Console.WriteLine("Обозначения: n k m -стороны треугольника, a b - углы, S - площадь, P - периметр");
+            Console.ReadKey();
+
+            var n = double.Parse(Console.ReadLine());
+            var a = double.Parse(Console.ReadLine());
+            var b = double.Parse(Console.ReadLine());
+            
+            double S = (n*n) / 2 * (Math.Sin(a*180/Math.PI) * Math.Sin(b*180/Math.PI)) / Math.Sin(180 - (a*180/Math.PI + b*180/Math.PI));
```

Comment:
Вы не разобрались с единицами измерения углов. В этом выражении вы из градусов вычитаете радианы
```suggestion
            double S = (n*n) / 2 * (Math.Sin(a*180/Math.PI) * Math.Sin(b*180/Math.PI)) / Math.Sin(**180 - (a*180/Math.PI + b*180/Math.PI**));
```